recipes/maturin/meta.yaml

```diff
@@ -0,0 +1,44 @@
+{% set name = "maturin" %}
+{% set version = "0.7.7" %}
+
+package:
+  name: "{{ name|lower }}"
+  version: "{{ version }}"
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: 2533c648aa8fa19f7a9d4f62282b4adce2f9c07c49f512005f92911be767555e
+
+build:
+  number: 0
+  skip: True  # [py27]
+  missing_dso_whitelist:          # [osx]
+    - /usr/lib/libresolv.9.dylib  # [osx]
+    - /usr/lib/libpam.2.dylib     # [osx]
```

Comment:
Do you need this line?

Reply:
I copied it by mistake. Just removed it.

Reply:
Thank you!